haptools/__main__.py

```diff
@@ -498,3 +498,48 @@ def transform(
 if __name__ == "__main__":
     # run the CLI if someone tries 'python -m haptools' on the command line
     main(prog_name="haptools")
+
+
+
+
+############ Haptools index ###############
+
+@main.command(short_help="Sort and index .hap files")
+@click.argument("haplotypes", type=click.Path(exists=True, path_type=Path))
+@click.option(
+    "-o",
+    "--output",
+    type=click.Path(path_type=Path),
+    default= None,
+    show_default="stdout",
```

Comment:
```suggestion
    show_default="path to the input file",
```
This way, we can tell them that the default output file is essentially the same as the input file path, except with ".gz" appended if it isn't already